controllers/workspace/provision/deployment.go

```diff
@@ -107,6 +109,35 @@ func SyncDeploymentToCluster(
 		}
 	}
 
+	// Get pod and update dw status on crashloopbackoff
+	podList, err := getPods(workspace, clusterAPI.Client)
+	if err != nil {
+		return DeploymentProvisioningStatus{
+			ProvisioningStatus: ProvisioningStatus{
+				Requeue: true,
+				Err:     err,
```

Comment:
If `err != nil`, the requeue is unnecessary (returning an error always triggers a requeue).

Reply:
Ah yes! Sorry, removed the Requeue! Good catch!